azure/packages/azure-client/package.json

```diff
@@ -85,7 +85,7 @@
     "typescript": "~4.5.5"
   },
   "peerDependencies": {
-    "fluid-framework": "^1.3.3"
+    "fluid-framework": "^1.3.5-121405"
```

Comment:
Still have to manually bump this? That bug is with Sonali in this sprint?

Reply:
Yeah, @sonalideshpandemsft looking at the bug.